CONTRIBUTING.md

```diff
@@ -1,94 +1,125 @@
 
-**Step 2:** Fork the `/dotnet/core-docs` repo.
+If it's a new topic, you can use [our template file](template.md) as your starting point. It contains our writing guidelines and also explains the metadata required for each article, such as author information.
 
-**Step 3:** Create a `branch` for your article.
+Place the article in the appropriate folder and any needed images in a media folder located in the same folder as the article.
 
-**Step 4:** Write your article, placing the article in its own folder and any
-needed images in a _static folder located in the same folder as the article.
-Be sure to follow the proper Markdown syntax. If you have code samples,
-place them in a folder within the `/samples/` folder.  For writing guidelines, see
-[our style guide](/styleguide.md). Also do not forget to remove the "wrench" icon 
-(🔧) from the TOC and the file heading, if applicable. 
+Be sure to follow the proper Markdown syntax.
+ 
+If you have code samples, place them in a folder within the `/samples/` folder. 
+Also do not forget to remove the "wrench" icon (🔧) from the TOC and the file heading, if applicable. 
 
-**Step 5:** Submit a Pull Request from your branch to `dotnet/core-docs/master`.
+**Step 5:** Submit a PR from your branch to `dotnet/core-docs/master`.
 
-**Step 6:** Discuss the Pull Request with the .NET team; make any requested
-updates to your branch. When they are ready to accept the PR, they will add a
-"LGTM" (Looks Good To Me) comment and any other steps that are (maybe) needed.
+If your PR is addressing an existing issue, add the `Fixes #Issue_Number` keyword to the commit message or PR description, so the issue can be automatically closed when the PR is merged. For more information, see [Closing issues via commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/).
 
-## Guidance checklist ##
+The .NET team will review your PR and let you know if the change looks good or if there are any other updates/changes necessary in order to approve it.
 
-Below is a short list of guiding rules that you should keep in mind when you are
-contributing to .NET Core documentation.
+**Step 6:** Make any necessary updates to your branch as discussed with the team. 
 
-- Don't forget to submit an issue before starting work on an article
-- Don't forget to create a separate branch before working on your article
-- Don't update or `merge` your branch after you submit your pull request
-- If updating code samples in `/samples/`, be sure any line number references
-	in your article remain correct
+When we are ready to accept the PR, we will let you know that the change looks good and merge it into the master branch. The master branch is our staging environment. 
 
-## Building the docs ##
+On a certain cadence, we push all commits from master branch into the live branch (our production environment) and then you'll be able to see your contribution live at https://docs.microsoft.com/dotnet/. 
```

Comment:
Slight subtlety ... the live branch itself is not "our production environment". Here, I'd say "the live Microsoft docs" or something like that. Very simply, there is ".NET Foundation" and then "Microsoft". .NET Core on GitHub (including docs) is a .NET Foundation thing and the distros (including docs) is a Microsoft thing.